lua/kubectl/utils/events.lua

```diff
@@ -68,13 +66,16 @@ function M.ColorStatus(status)
     AlreadyMountedVolume = true,
     Available = true,
     ContainerCreating = true,
+    Degraded = true,
```

Comment:
The not ready ones I understand but degraded only warning?

Reply:
sure thing, moved it to errorStatuses